src/python/pants_release/generate_github_workflows.py

```diff
@@ -1259,6 +1259,19 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Trigger docs sync",
+                    "env": {
+                        "GH_TOKEN": "${{ secrets.WORKER_PANTS_PANTSBUILD_ORG_TRIGGER_PAT }}",
```

Comment:
@benjyw let's work on making this PAT and testing it

Reply:
This PAT now exists in the pantsbuild/pants secrets, with read/write access to workflows on pantsbuild/pantsbuild.org. let me know if the permissions are not sufficient.

Reply:
That sounds sufficient. I'll test it tonight/tomorrow

Reply:
The PAT Benjy sent me works! :tada: